src/infrastructure/transaction/CreateTransactionFromDTO.ts

```diff
@@ -439,7 +441,7 @@ const CreateStandaloneTransactionFromDTO = (transactionDTO, transactionInfo): Tr
         return new MosaicMetadataTransaction(
             transactionDTO.network,
             transactionDTO.version,
-            extractDeadline(transactionDTO.deadline),
+            extractDeadline(nemesisEpoch, transactionDTO.deadline),
```

Comment:
If deadline stores deadline from server as-as, you don't need to provide the nemesisEpech when mapping